packages/airnode-validator/src/validators/validators.test.ts

```diff
@@ -195,9 +195,7 @@ describe('validators integration test', () => {
         keyFormattingMessage('myPath', ['paths', 'myPath']),
         error('paths.myPath.get : allowed methods are only "get" or "post" not "get "'),
         error("Allowed values in paths.myPath.get .parameters.[0].in are: 'path', 'query', 'header' or 'cookie'"),
-        keyFormattingMessage('mySecurityScheme ', ['components', 'securitySchemes', 'mySecurityScheme ']),
```

Comment:
These changes in the test are because of the OIS being updated from pre-alpha to v1.0, right?

Reply:
Yes, I would say so.